bibliopixel/layout/layout.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, drivers, threadedUpdate, brightness,
         self.animation_sleep_time = None
 
         self.threading = UpdateThreading(threadedUpdate, self)
+        self.brightness = brightness
```

Comment:
There was nothing in layout storing this... only in Drivers. I honestly don't care what it is in the drivers, but I care about what the layout *thinks* it is.